Tools/parametric_model/src/models/dynamics_model.py

```diff
@@ -609,161 +628,184 @@ def plot_scatter(ax, title, dataframe_x, dataframe_y, dataframe_z, color='blue')
-                    "det": float(np.linalg.det(information_matrix_f)/self.n_samples)
+                {
+                    "lin": {
+                        "trace": float(np.trace(information_matrix_f)/self.n_samples),
+                        "min_eig": float(min(np.abs(np.linalg.eigvals(information_matrix_f)))/self.n_samples),
+                        "inv_cond": float(min(np.abs(np.linalg.eigvals(information_matrix_f))) /
+                                          max(np.abs(np.linalg.eigvals(information_matrix_f))) / self.n_samples),
+                        "det": float(np.linalg.det(information_matrix_f)/self.n_samples)
+                    }
                 }
-            }
             )
 
         if self.estimate_moments:
 
             self.fisher_metric["FIM"].update(
-            {
-                "rot":{
-                    "trace":float(np.trace(information_matrix_m))/self.n_samples,
-                    "min_eig":float(min(np.abs(np.linalg.eigvals(information_matrix_m)))/self.n_samples),
-                    "inv_cond":float(min(np.abs(np.linalg.eigvals(information_matrix_m))) / \
-                        max(np.abs(np.linalg.eigvals(information_matrix_m)))/self.n_samples),
-                    "det":float(np.linalg.det(information_matrix_m)/self.n_samples)
+                {
+                    "rot": {
+                        "trace": float(np.trace(information_matrix_m))/self.n_samples,
+                        "min_eig": float(min(np.abs(np.linalg.eigvals(information_matrix_m)))/self.n_samples),
+                        "inv_cond": float(min(np.abs(np.linalg.eigvals(information_matrix_m))) /
+                                          max(np.abs(np.linalg.eigvals(information_matrix_m)))/self.n_samples),
+                        "det": float(np.linalg.det(information_matrix_m)/self.n_samples)
+                    }
                 }
-            }
             )
 
-
-            ## min eigenvalue
+            # min eigenvalue
             # fisher_information_f_mat[i]= min(np.abs(np.linalg.eigvals(fisher_information_matrix_f)))
             # fisher_information_m_mat[i]= min(np.abs(np.linalg.eigvals(fisher_information_matrix_m)))
 
-            ## trace
+            # trace
             # fisher_information_f_mat[i]= np.trace(fisher_information_matrix_f)
             # fisher_information_m_mat[i]= np.trace(fisher_information_matrix_m)
 
-            ## condition number
+            # condition number
             # fisher_information_f_mat[i]= min(np.abs(np.linalg.eigvals(fisher_information_matrix_f))) / \
             # max(np.abs(np.linalg.eigvals(fisher_information_matrix_f)))
             # fisher_information_m_mat[i]= min(np.abs(np.linalg.eigvals(fisher_information_matrix_m))) / \
-            # max(np.abs(np.linalg.eigvals(fisher_information_matrix_m)))
\ No newline at end of file
+            # max(np.abs(np.linalg.eigvals(fisher_information_matrix_m)))
```

Comment:
The changes in this file seems to be format fixes. Could we make this a separate PR so it would be easier to find out what changed?

Reply:
Removed formatting changes - the file was not changed at all, formatting was done by an auto-formatter